aip_x1_launch/launch/velodyne_node_container.launch.py

```diff
@@ -106,13 +106,13 @@ def create_parameter_dict(*args):
 
     nodes.append(
         ComposableNode(
-            package="velodyne_pointcloud",
-            plugin="velodyne_pointcloud::Interpolate",
-            name="velodyne_interpolate_node",
+            package="pointcloud_preprocessor",
+            plugin="pointcloud_preprocessor::DistortionCorrectorComponent",
+            name="distortion_corrector_node",
             remappings=[
-                ("velodyne_points_ex", "self_cropped/pointcloud_ex"),
-                ("velodyne_points_interpolate", "rectified/pointcloud"),
-                ("velodyne_points_interpolate_ex", "rectified/pointcloud_ex"),
+                ("~/input/velocity_report", "/vehicle/status/velocity_status"),
+                ("~/input/pointcloud", "mirror_cropped/pointcloud_ex"),
+                ("~/output/pointcloud", "rectified/pointcloud_ex"),
```

Comment:
Why is this file in x1?
Should it be merged common_sensor_launch/launch/velodyne_node_container.launch.py?

Reply:
In aip_x1_launch, there is no mirror_crop_filter because x1 reference has no mirror so far and I fixed bug in aip_x1_launch https://github.com/tier4/aip_launcher/pull/24/commits/e402574426a5cfb999df7c2187c1d64e0fed35a7

Reply:
How about making the argument which whether or not to include mirror crop?

Reply:
It seems good. I'll fix it in new pull request.

Reply:
Thank you!
I apologize for asking for so many additional things.

Reply:
@yukkysaito Either abolishingmirror crop filter or creating use_mirror_crop argument will be supported other pull request. Could you resolve conversation?

Reply:
okay :+1: